Ultz.Online/Core/Net/UI.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Linq;
+
+namespace Ultz.Online.Core.Net
+{
+    public static class UI
+    {
+        public static void SendSidenav(Guid guid, string[] nav)
+        {
+            Gcm.As(Gcm.ExecuteOnAllMastersAsync("ui::sendSidenav", new object[] { guid, nav }).Do());
+        }
+
+        public static void SendContent(Guid guid, string[] content)
+        {
+            Gcm.As(Gcm.ExecuteOnAllMastersAsync("ui::sendContent", new object[] { guid, content }).Do());
+        }
+
+        public static bool RegisterURL(string url, string second)
+        {
+            return Gcm.As<bool>(Gcm.ExecuteOnAllMastersAsync("ui::registerURL", new[] { url, second }).Do()).Any(x => x);
+        }
+
+        public static bool UnregisterURL(string url, string second)
+        {
+            return Gcm.As<bool>(Gcm.ExecuteOnAllMastersAsync("ui::unregisterURL", new[] { url, second }).Do()).Any(x => x);
+        }
+
+        public static bool RegisterEvent(string ev)
+        {
+            return Gcm.As<bool>(Gcm.ExecuteOnAllMastersAsync("ui::registerEvent", new[] { ev }).Do()).Any(x => x);
+        }
+
+        public static bool UnregisterEvent(string ev)
+        {
+            return Gcm.As<bool>(Gcm.ExecuteOnAllMastersAsync("ui::unregisterEvent", new[] { ev }).Do()).Any(x => x);
+        }
+
+        public static void Notify(string[] notify, int Serverity, string som1, string som2) //?????? No clue what I had to call these methods
```

Comment:
Don't worry, it took me a while to remember what these were meant to be also haha.
```suggestion
        public static void Notify(string[] usersOrGroups, int serverity, string title, string content)
```